modules/core/core-api/src/main/java/com/enonic/xp/content/ProjectSynchronizer.java

```diff
@@ -0,0 +1,9 @@
+package com.enonic.xp.content;
+
+public interface ProjectSynchronizer
+{
+    void syncContent( final ProjectContentSyncParams params );
+
+    void syncEvent( final ProjectEventSyncParams params );
```

Comment:
this is confusing. Sounds like this method will sync an event?

Reply:
could be both called `sync`?